### ISSUE TITLE
Stable crashes (v31.3 and earlier)

### ISSUE THREAD

**Issue by winterwolf** [miki151/keeperrl#1724]
I check for updates of your game every new version and it is still not playable.

Here I recorded the video:

https://youtu.be/nQeL1Npxl9M

(pause it at the end if you want to see logs and system info)

Steps to reproduce:

1. download free version or build from source (any branch)
2. launch the game
3. press start, wait ~1 minute
4. get crash

**Comment by GameDungeon** [comment]
I could be wrong, but the crash seems to stem from being unable to connect to steam. How are you building keeper?

**Comment by winterwolf** [comment]
@GameDungeon I don't have steam and don't want to have it. I downloaded free version from itch.io, not from steam. When I tried to build the game, I used flag `NO_STEAMWORKS=true`. I think steam is not the reason because I can launch exe file through wine and it is works fine without steam. But wine is not stable itself. I want to play native game. And yes, I checked, if you simple ignore retired dungeons and don't add them, game still crashes. It needs to exactly disable online feature in settings to work.

**Comment by winterwolf** [comment]
@GameDungeon you see what the problem... **I can** download retired dungeons, **I can** start the game with them, **I can** download and enable any mod. All online features works! but game crashes :/

**Comment by miki151** [comment]
What do you mean by "curl 4 in the libs directory"? If you're building from source then the game should use your system's libraries.

Does the prebuilt free version also crash?

**Comment by GameDungeon** [comment]
Would the prebuilt work on manjaro?

**Comment by winterwolf** [comment]
> What do you mean by "curl 4 in the libs directory"? If you're building from source then the game should use your system's libraries.
> 
> Does the prebuilt free version also crash?

Sorry, I don't understand what is prebuilt version. Where can I get it?

I downloaded my current game version here: https://miki151.itch.io/keeperrl ("Download demo" button at the bottom).

This version contains directory called `lin64` with `libcurl-gnutls.so.4`.

**Comment by winterwolf** [comment]
WOW!!! :octopus: I just removed this file and game not crashes anymore! Then I restore it and game crashes again. So I was right! Can't believe it! Your curl is outdated.

**Comment by GameDungeon** [comment]
that is the prebuilt one. If it works, why would you need an update

**Comment by GameDungeon** [comment]
Hey Michal, maybe update curl, but other than that is can be closed

**Comment by miki151** [comment]
Was the version that crashed built from source or downloaded?

**Comment by Zireael07** [comment]
> I downloaded my current game version here: https://miki151.itch.io/keeperrl ("Download demo" button at the bottom).

I guess it was downloaded?

**Comment by miki151** [comment]
They mentioned: "download free version or build from source (any branch)"

This kind of issue happens if the binary is built with a different library version than the one supplied, for example if you copy your own build into a downloaded version. That's why I want to clarify.

**Comment by winterwolf** [comment]
@miki151 yes, this issue happens with prebuilded version, but last time when I tried to build the game by yourself, I get very similar error:  #1709. Probably the reason for that bug is something else. I can not say for sure.
